### DISCUSSION TITLE
Eigenwirtschaftlicher Verkehr

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#328] [Ideas]
Man könnte selbst die Strecken aussuchen und fahren, das Einkommen hängt aber von der Fahrgastanzahl ab.

Die wiederum wäre abhängig von den angefahrenen Bahnhöfen, bzw. der Nachfrage dort (bzw. quasi der wartenden Reisenden).
Die Nachfrage würde kontinuierlich, je nach Bahnhofskategorie langsam oder schnell ansteigen, ggf. mit einem Maximum.
Wenn ein (eigenwirtschaftlicher) Zug da ankommt, steigen, abhängig von Kategorie und Nachfrage, Fahrgäste ein/aus und die Nachfrage wird um die Anzahl der eingestiegenen Fahrgäste reduziert.
Dabei sollten niemals alle Reisende an einer Station gleichzeitig in einen einzelnen Zug steigen, sondern immer nur ein Teil, damit nicht so ein Großkonzern einen irrsinnig langen Dosto nach Berlin schickt und alles aufsammelt.

Um zu vermeiden, dass dann einfach _jeder_ Hp auf einer Route angefahren wird, müssten hier größenabhängig Stationspreise (zusätzlich zu den Trassengebühren) eingeführt werden, die für die Stationem, an denen gehalten wird, berechnet werden.

Die km, die dadurch gefahren würden, sollten nicht (oder getrennt) auf das Ranking der EVUs angerechnet werden, um extreme Langläufer zu verhindern.

Hinweis: Ich weiß selbst noch nicht ganz, ob das eine sinnvolle Idee ist oder einfach nur noch mehr Geld in die Kassen des Flauschehorns spült 😂
Vor allem müsste es auch ein Balancing geben, dass nicht einfach immer die längste Strecke Sinn ergibt. Vielleicht ließe sich auch ein Limit an aktiven eigenwirtschaftlichen Fahrten einführen

**Comment by marhei** [comment]
Klingt interessante, sobald ich Luft hab mach ich mir da mal Gedanken.
